README.md

```diff
@@ -33,10 +33,10 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [Awesome books] <a name="about-project"></a>
+# 📖 [LeaderBoard] <a name="about-project"></a>
 
 
-**WebPack and Linters Template** This is a template Repository for webpack project with Linters pre-installed
+**LeaderBoard** The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
 
```

Comment:
- [ ] Live demo link in your ReadMe on _line 64_ is broken, kindly endeavor to fix or remove it if this app is not yet deployed.

**Clicking on the Live demo link in your ReadMe..**
![image](https://user-images.githubusercontent.com/12745474/219909597-5effbd5c-69cc-42f4-b7e5-fc37a396a24b.png)

**Leads to a 404 request error page:**
![image](https://user-images.githubusercontent.com/12745474/219909645-acca4d56-901d-43fd-9b49-1d6640bd58fd.png)

Reply:
Okay @rloterh Thank you for the feedback. Just done it